config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -35,17 +43,71 @@ spec:
         ansiblePort: 22
         ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
         ansibleVars: |
+          service_net_map:
+            nova_api_network: internal_api
+            nova_libvirt_network: internal_api
+
+          # edpm_network_config
+          # Default nic config template for a EDPM compute node
+          # These vars are edpm_network_config role vars
           edpm_network_config_template: templates/net_config_bridge.j2
           edpm_network_config_hide_sensitive_logs: false
+          #
+          # These vars are for the network config templates themselves and are
+          # considered EDPM network defaults.
           neutron_physical_bridge_name: br-ex
           neutron_public_interface_name: eth0
+          ctlplane_mtu: 1500
+          ctlplane_subnet_cidr: 24
+          ctlplane_gateway_ip: 192.168.122.1
+          ctlplane_host_routes:
+          - ip_netmask: 0.0.0.0/0
+            next_hop: 192.168.122.1
+          external_mtu: 1500
+          external_vlan_id: 44
+          external_cidr: '24'
+          external_host_routes: []
+          internal_api_mtu: 1500
+          internal_api_vlan_id: 20
+          internal_api_cidr: '24'
+          internal_api_host_routes: []
+          storage_mtu: 1500
+          storage_vlan_id: 21
+          storage_cidr: '24'
+          storage_host_routes: []
+          tenant_mtu: 1500
+          tenant_vlan_id: 22
+          tenant_cidr: '24'
+          tenant_host_routes: []
+          role_networks:
+          - InternalApi
+          - Storage
+          - Tenant
```

Comment:
I think we might need External here

Reply:
done!

Reply:
https://github.com/openstack-k8s-operators/install_yamls/pull/172